<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>
<title>landing_page</title>
</head>
<body><h1 id='rabbit-gastrulation-web-portal'>Rabbit gastrulation web portal</h1>
<p>This website is still under active development.</p>
<p>To report any bugs/issues please contact Daniel Keitley (<a href='mailto:dk562@cam.ac.uk'>dk562@cam.ac.uk</a>) or file an issue on the <a href='https://github.com/dkeitley/scrabbit-shiny'>Github repository</a>.</p>
<p>Please note that some visualisations may take a bit of time to load. </p>
<hr />
<p>&nbsp;</p>
<h3 id='pages'>Pages</h3>
<h5 id='vitessce'>Vitessce</h5>
<p>This page utilises the powerful visualisation tool, <a href='http://vitessce.io/'>Vitessce</a>, to display our single-cell transcriptomics data. It allows you to query gene expression and explore information about each cell in various reduced dimensional spaces. </p>
<h5 id='genome-annotation'>Genome annotation</h5>
<p>On this page we use the <a href='https://igv.org/'>Integrative Genomics Viewer (IGV)</a> to display our edited rabbit transcriptome and the read coverage for an individual GD8 sample from our 10X single-cell RNA-seq data.  This can used to assess the annotation quality for particular genes of interest, such as in cases where no expression is observed in the processed dataset.</p>
<p>See the &#39;<em>Improving transcriptome mapping</em>&#39; section of the paper methods for more information on how the modified rabbit transcriptome annotation was produced. </p>
<p>&nbsp;</p>
<h3 id='other-links'>Other links</h3>
<p><a href='https://www.ebi.ac.uk/research-beta/marioni/'>Marioni lab website</a></p>
<p>&nbsp;</p>
<h3 id='contact-us'>Contact us</h3>
<p>We would appreciate any feedback you have. If there are particular visualisations you would like us to create, please let us know via <a href='mailto:dk562@cam.ac.uk'>email</a> or the <a href='https://github.com/dkeitley/scrabbit-shiny'>github repository</a>.</p>
</body>
</html>