# Rabbit gastrulation web portal

This website is still under active development.

To report any bugs/issues please contact Daniel Keitley (<a href='mailto:dk562@cam.ac.uk'>dk562@cam.ac.uk</a>) or file an issue on the <a href='https://github.com/dkeitley/scrabbit-shiny'>Github repository</a>.

Please note that some visualisations may take a bit of time to load. 

------



### Pages

##### Vitessce

This page utilises the powerful visualisation tool, [Vitessce](http://vitessce.io/), to display our single-cell transcriptomics data. It allows you to query gene expression and explore information about each cell in various reduced dimensional spaces. 

##### Genome annotation

On this page we use the [Integrative Genomics Viewer (IGV)](https://igv.org/) to display our edited rabbit transcriptome and the read coverage for an individual GD8 sample from our 10X single-cell RNA-seq data.  This can used to assess the annotation quality for particular genes of interest, such as in cases where no expression is observed in the processed dataset.

See the '*Improving transcriptome mapping*' section of the paper methods for more information on how the modified rabbit transcriptome annotation was produced. 



### Other links

[Marioni lab website](https://www.ebi.ac.uk/research-beta/marioni/)



### Contact us

We would appreciate any feedback you have. If there are particular visualisations you would like us to create, please let us know via <a href='mailto:dk562@cam.ac.uk'>email</a> or the <a href='https://github.com/dkeitley/scrabbit-shiny'>github repository</a>.